searxng_extra/update/update_languages.py

```diff
@@ -24,15 +32,15 @@
 def fetch_supported_languages():
     set_timeout_for_thread(10.0)
 
-    engines_languages = dict()
+    engines_languages = {}
     names = list(engines)
     names.sort()
 
     for engine_name in names:
         if hasattr(engines[engine_name], 'fetch_supported_languages'):
             engines_languages[engine_name] = engines[engine_name].fetch_supported_languages()
             print("fetched %s languages from engine %s" % (len(engines_languages[engine_name]), engine_name))
-            if type(engines_languages[engine_name]) == list:
+            if type(engines_languages[engine_name]) == list:  # pylint: disable=unidiomatic-typecheck
```

Comment:
any reason not to do `isinstance(engines_languages[engine_name], list)`?

Reply:
not really .. since this commit just a pylint check, I wont change the code if it is not really needed.  That's why I most often add only a `pylint: disable=`.

Reply:
Well, you did rename some variables ... it seems a bit arbitrary where you draw the line.

Reply:
>>  wont change the code if it is not really needed.

> it seems a bit arbitrary where you draw the line.

Yes that true.  Most often I change stuff I know to 100% that there is no functional change.  In this case `type()` is a different function call compared to a `instance()` with a different prototype ... that is why I hesitate to change .. but as you said it is subjective decision.

If you insist, I can change it.

Reply:
Nah it's fine.